composer.json

```diff
@@ -32,15 +32,15 @@
             " bash update-version.sh" 
         ],
         "test-setup":[ 
-            "bash bin/install-wp-tests.sh wp_tests_db root '' 127.0.0.1 latest" 
+            "bash vendor/bcgov/wordpress-utils/bin/install-wp-tests.sh wordpress_test root 'rootpassword' 0.0.0.0 latest"
         ],
         "production": [
             "composer validate",
-            "Bcgov\\Script\\Checklists::postProductionChecks"
-        ],
-        "checklist": [
-            "composer validate",
-            "Bcgov\\Script\\Checklists::postProductionChecks"
+            "composer phpcs",
```

Comment:
i wonder if there is a way to ensure that the npm build is production done, as this sometimes has caused issues, where a dev build has been sent to production... maybe add it to future todos.